packages/palette_generator/test/palette_generator_test.dart

```diff
@@ -2,6 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// TODO(dit): Reenable web tests, https://github.com/flutter/flutter/issues/151498
+@TestOn('vm') // Fails on CI with canvaskit on the web
+library;
```

Comment:
I think this file's changes can be reverted now.

Reply:
Oh, I thought the config file would stop it from getting this far. If it's still useful, feel free to leave it.

Reply:
Actually, let me double check what `flutter test` does when the config file is present, I thought I saw it run the test anyway but it doesn't hurt to verify!

Reply:
```console
$ flutter test --platform=chrome --web-renderer=canvaskit
Warning: this package doesn't support running tests on browsers.

No tests ran.
No tests were found.

$ echo $?
1
```

OK, that return code of 1 should be fine in local development. Cleaning up the test file!

Reply:
> ```
> $ echo $?
> 1
> ```

Ah right, I remember this now. That's why I had to build special tool logic to parse that file and pre-filter in CI :|